### PR TITLE
Add page adhesion

### PR DESCRIPTION
**Pull request by marineb**
- [x] Registration - Main
- [x] Registration - Donation
- [x] Registration - Interests
- [x] Registration - Comités

### CODE REVIEW

File: app/Resources/views/page/adherer.html.twig [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html.twig' %}
+
```
Comment (HeahDude) [inline app/Resources/views/page/adherer.html.twig]: Duplicate of `https://github.com/EnMarche/en-marche.fr/blob/master/app/Resources/views/membership/register.html.twig`?

File: src/AppBundle/Controller/PageController.php [hunk 2]
```diff
@@ -148,4 +148,13 @@ public function botTestingAction()
 
         return $this->render('page/bot-testing.html.twig');
     }
+
+    /**
+     * @Route("/adherer", name="page_adherer")
```
Comment (HeahDude) [inline src/AppBundle/Controller/PageController.php]: Duplicate of `https://github.com/EnMarche/en-marche.fr/blob/master/src/AppBundle/Controller/MembershipController.php#L28`?

Comment (tgalopin) [inline src/AppBundle/Controller/PageController.php]: Yes but it's not the same aim, it's a page for the legacy subscription.

**Comment by tgalopin** [comment]
Is this really ready to merge? I dont see the background image.

**Comment by marineb** [comment]
Pardon, je n'ai pas bien QAed. J'y retourne.

**Pull request merged**